tests/scanners/test.javascript.js

```diff
@@ -47,6 +67,41 @@ describe('JavaScript Scanner', () => {
     expect(jsScannerWithOptions.options.foo).toEqual('bar');
   });
 
+  it('should be initialised with empty excluded rules object', () => {
+    const jsScanner = new JavaScriptScanner('', 'filename.txt');
+    expect(typeof jsScanner.disabledRules).toEqual('object');
+    // This test assures us the disabledRules can be accessed like an object.
+    expect(typeof jsScanner.disabledRules.someUndefinedProp).toEqual('undefined');
+  });
+
+  it('should be initialised with disabledRules from options', () => {
+    const jsScanner = new JavaScriptScanner('', 'filename.txt', {
+      disabledRules: 'no-eval, no-implied-eval,                 no-unsafe-innerhtml/no-unsafe-innerhtml',
+    });
+    expect(typeof jsScanner.disabledRules).toEqual('object');
+    // This test assures us the disabledRules built properly.
+    expect(jsScanner.disabledRules['no-unsafe-innerhtml/no-unsafe-innerhtml']).toBe(true);
+  });
+
+  it('should be initialised with empty excluded rules object, when there is no string', () => {
+    const jsScanner = new JavaScriptScanner('', 'filename.txt', {
+      disabledRules: true,
+    });
+    expect(typeof jsScanner.disabledRules).toEqual('object');
+    expect(jsScanner.disabledRules).toEqual({});
+  });
```

Comment:
Ah, here you did the check correctly. I don't think you need the `typeof` check though.